src/slack-send.js

```diff
@@ -128,9 +131,11 @@ module.exports = async function slackSend(core) {
 
       try {
         await axios.post(webhookUrl, payload, axiosOpts);
+        console.log(JSON.stringify(payload));
       } catch (err) {
         console.log('axios post failed, double check the payload being sent includes the keys Slack expects');
-        console.log(payload);
+        // console.log(payload);
+        console.log(JSON.stringify(payload));
```

Comment:
It might be a good time to remove the commented logs and perhaps enhance the outputs in this section with `console.error`?


```suggestion
        console.error(`Attempted to POST payload: ${JSON.stringify(payload)}`);
```